birdhouse/env.local.example

```diff
@@ -251,7 +235,33 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #                                   }
 #"
 
+# Load jobs to automatically deploy the custom notebooks from the specific images
+#
+# Ensure we always use the "latest" version of the "cronjob generation code"
+# Path to a checked out repo of "pavics-jupyter-base" which contains the config required for the cronjob generation
+#CHECKOUT_PAVICS_CONFIG="/path/to/checkout/pavics-jupyter-base"
```

Comment:
CHECKOUT_PAVICS_CONFIG ... I think a different var name is more appropriate, this is very generic.

Reply:
Put the github link to pavics-jupyter-base so user can perform the checkout without guessing/searching.